spec/system/feedback_spec.rb

```diff
@@ -23,12 +23,18 @@
         click_on "Continue to download and finish"
         expect(page).to have_content("Were you satisfied with this service?")
         click_on "Yes"
+        # TODO:  temp fix for parallel running
```

Comment:
Question - is this a temp fix for when we get to [EL-1374](https://dsdmoj.atlassian.net/browse/EL-1374)?

Reply:
Yes, this is currently a problem anyway, but when running lots of processes (10 x browsers and 10 rspecs on my M1 machine by default) it happens so often that a clean build is almost impossible to create